talonscript-mode.el

```diff
@@ -54,7 +54,7 @@
 
    ;; Matches keys that are function calls, e.g will match "action" in:
    ;; action(edit.zoom_in): key(ctrl-+)
-   (cons "^\\([a-zA-Z0-9_-.]*\\)(" '(1 font-lock-type-face t))
```

Comment:
I think this is a mistake in the original? If not, I don't understand what it is supposed to do.